### PR TITLE
fixed setting secrets.credentialsSecretName doesn't do anything

### PR DESCRIPTION
**Pull request by this-is-r-gaurav**
Fix for #372

### CODE REVIEW

File: charts/timescaledb-single/templates/statefulset-timescaledb.yaml [hunk 1]
```diff
@@ -255,10 +255,10 @@ spec:
         {{- end }}
         envFrom:
         - secretRef:
-            name: {{ template "secrets_credentials" . }}
+            name: {{- if .Values.secrets.credentialsSecretName }} "{{ .Values.secrets.credentialsSecretName }}" {{- else }} {{ template "secrets_credentials" . }} {{- end}}
             optional: false
         - secretRef:
-            name: {{ template "secrets_pgbackrest" . }}
+            name: {{- if .Values.secrets.pgbackrestSecretName }} "{{ .Values.secrets.pgbackrestSecretName }}" {{- else }} {{ template "secrets_pgbackrest" . }} {{- end}}
```
Comment (graveland) [inline charts/timescaledb-single/templates/statefulset-timescaledb.yaml:261]: Couldn't you put this logic into the secrets_X templates in templates/_helpers.tpl? It'd keep it future-proof and make sure all uses of the secrets would pull from the configured XSecretName values then.

**Comment by paulfantom** [comment]
It seems that a fix for this is also being worked on in https://github.com/timescale/timescaledb-kubernetes/pull/359

File: charts/timescaledb-single/templates/statefulset-timescaledb.yaml (hunk 1 above)
Comment (this-is-r-gaurav) [reply charts/timescaledb-single/templates/statefulset-timescaledb.yaml:261]: ok it can be done

**Comment by this-is-r-gaurav** [comment]
> It seems that a fix for this is also being worked on in #359

Do you want me to close the PR?

**Comment by this-is-r-gaurav** [comment]
> It seems that a fix for this is also being worked on in #359

That was a partial fix, there were some more issue that may break chart due to | quote he is using i addressed this in my recent commit.

**Review by graveland — APPROVED** [review]
Looks good to me!

**Pull request merged**